src/gei/Commands/MigrateRepo/MigrateRepoCommandArgs.cs

```diff
@@ -41,11 +43,26 @@ public override void Validate(OctoLogger log)
             DefaultSourcePat(log);
             DefaultTargetRepo(log);
 
+            if (GitArchiveUrl.HasValue() && GitArchiveFilePath.HasValue())
+            {
+                throw new OctoshiftCliException("The options --git-archive-url and --git-archive-file-path may not be used together");
+            }
+
+            if (MetadataArchiveUrl.HasValue() && MetadataArchiveFilePath.HasValue())
+            {
+                throw new OctoshiftCliException("The options --metadata-archive-url and --metadata-archive-file-path may not be used together");
+            }
```

Comment:
Are we going to support a scenario where `GitArchiveUrl` is provided with `MetadataArchivePath` or vice versa? if not then we need to validate those scenarios too.

Reply:
Probably not but the combination of MetadataArchiveUrl requiring GitArchiveUrl but also not allowing the path settings currently prevent it.